data/campaigns/Delfadors_Memoirs/scenarios/15_Save_the_King.cfg

```diff
@@ -447,6 +447,8 @@
         [/gold]
     [/event]
 
+    #TODO: add a death event for Zorlan
+
```

Comment:
Why not just write one?  A lame event will get fixed long before a missing one, even if marked as a TODO.

Reply:
I don't dare writing any prose as i'm not English, i'm not American and English is not my strong point. I can create a new issue, but i definitely won't write any prose, because i'm not able to

Reply:
Here's some bad prose, with translation hints:

```
# po: "rout" isn't a misspelling, it's an archaic term for defeating an army
# po: There's no check for orcs with these names to be on the map, but they're all named after the warring chieftains after Kalenz and Landar defeated the Horde in LoW
"Khrubar, Gvur, Ozul, one of you must lead the Horde, or the elves will rout our separate tribes again!"
```

Reply:
Thanks! :)